### PR TITLE
Add alias _mtime for virtual column _time to file alike storages

### PR DESCRIPTION
**Pull request by ilejn**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
_time virtual column for file alike storages is also known as _mtime

Per https://github.com/ClickHouse/ClickHouse/pull/64947#issuecomment-2208980503

### CODE REVIEW

File: src/Storages/VirtualColumnUtils.cpp [hunk 1]
```diff
@@ -112,7 +112,7 @@ void filterBlockWithDAG(ActionsDAGPtr dag, Block & block, ContextPtr context)
 
 NameSet getVirtualNamesForFileLikeStorage()
 {
-    return {"_path", "_file", "_size", "_time"};
+  return {"_path", "_file", "_size", "_time", "_mtime"};
```
Comment (azat) [inline src/Storages/VirtualColumnUtils.cpp:115]:
```suggestion
    return {"_path", "_file", "_size", "_time", "_mtime"};
```

File: docs/en/sql-reference/table-functions/url.md [hunk 2]
```diff
@@ -54,6 +54,7 @@ Character `|` inside patterns is used to specify failover addresses. They are it
 - `_file` — Resource name of the `URL`. Type: `LowCardinalty(String)`.
 - `_size` — Size of the resource in bytes. Type: `Nullable(UInt64)`. If the size is unknown, the value is `NULL`.
 - `_time` — Last modified time of the file. Type: `Nullable(DateTime)`. If the time is unknown, the value is `NULL`.
```
Comment (azat) [inline docs/en/sql-reference/table-functions/url.md:56]: The idea was to make `_mtime` primary choice, so if going in this direction, it make sense to remove `_time` from documentation

**Comment by robot-clickhouse** [comment]
*This is an automated comment for commit 7eca366e40e315f3917c33077606de38441443a1 with description of existing statuses. It's updated for the latest CI running*

[❌ Click here](https://s3.amazonaws.com/clickhouse-test-reports/66123/7eca366e40e315f3917c33077606de38441443a1/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Style check</td><td>Runs a set of checks to keep the code style clean. If some of tests failed, see the related log from the report</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/66123/7eca366e40e315f3917c33077606de38441443a1/style_check.html">❌ failure</a></td></tr>
<tbody>
</table>

**Review by kssenii — APPROVED** [review]

**Comment by kssenii** [comment]
@ilejn could you please fix style check

**Comment by alexey-milovidov** [comment]
@ilejn We found a failed test, and you are the author. Let's fix it before merging this PR: #66998

**Comment by alexey-milovidov** [comment]
I don't understand why this is needed.

**Comment by alexey-milovidov** [comment]
We don't want to be TIMTOWTDI, and `time` is a better name.

**Pull request closed without merging**